packages/trackerless-network/src/logic/RandomGraphNode.ts

```diff
@@ -58,6 +62,8 @@ export interface StrictRandomGraphNodeConfig {
     nodeName: string
     acceptProxyConnections: boolean
     neighborUpdateInterval: number
+    inspector: IInspector
+    inspectingConnections: PeerList
```

Comment:
Would it make sense if we could query the connections from `inspector` and remove this field?

Reply:
moved to InspectionRpcServer